gluten-ut/spark35/src/test/scala/org/apache/spark/sql/hive/execution/GlutenHiveUDFSuite.scala

```diff
@@ -16,77 +16,31 @@
  */
 package org.apache.spark.sql.hive.execution
 
-import org.apache.gluten.execution.CustomerUDF
-
-import org.apache.spark.{SparkConf, SparkContext, SparkFunSuite}
-import org.apache.spark.internal.config
-import org.apache.spark.internal.config.UI.UI_ENABLED
-import org.apache.spark.sql.{GlutenTestsBaseTrait, QueryTest, SparkSession}
-import org.apache.spark.sql.catalyst.expressions.CodegenObjectFactoryMode
-import org.apache.spark.sql.catalyst.optimizer.ConvertToLocalRelation
-import org.apache.spark.sql.hive.{HiveExternalCatalog, HiveUtils}
-import org.apache.spark.sql.hive.client.HiveClient
-import org.apache.spark.sql.hive.test.TestHiveContext
-import org.apache.spark.sql.internal.SQLConf
-import org.apache.spark.sql.internal.StaticSQLConf.WAREHOUSE_PATH
-import org.apache.spark.sql.test.SQLTestUtils
-
-import org.scalatest.BeforeAndAfterAll
+import org.apache.gluten.execution.{ColumnarPartialProjectExec, CustomerUDF}
+
+import org.apache.spark.SparkConf
+import org.apache.spark.sql.Row
 
 import java.io.File
 
-trait GlutenTestHiveSingleton extends SparkFunSuite with BeforeAndAfterAll {
-  override protected val enableAutoThreadAudit = false
+import scala.collection.mutable
 
-}
+class GlutenHiveUDFSuite extends GlutenHiveSQLQuerySuiteBase {
```

Comment:
Can we move this test to `backends-velox/src/test`?
We expect the variants of Spark's tests are added in `spark-ut/sparkxx` folder. For other tests not depending on a specific Spark version, we may move it to a common test module.

Reply:
thanks. fixed